spec/classes/puppet_server_config_spec.rb

```diff
@@ -832,6 +836,29 @@
         end
       end
 
+      describe 'with server_custom_trusted_oid_mapping overwritten' do
+        let :pre_condition do
+          "class {'puppet':
+              server                            => true,
+              server_custom_trusted_oid_mapping => {
+                '1.3.6.1.4.1.34380.1.2.1.1' => {
+                  shortname => 'myshortname',
+                  longname  => 'My Long Name',
+                },
+                '1.3.6.1.4.1.34380.1.2.1.2' => {
+                  shortname => 'myothershortname',
+                },
+              }
+           }"
+        end
+
+        it 'should have a configured custom_trusted_oid_mapping.yaml' do
+          should contain_file("#{confdir}/custom_trusted_oid_mapping.yaml").
```

Comment:
We have a `verify_exact_contents` helper which might be easier to use.

Reply:
This is really helpful, thanks! :)